Gemfile

```diff
@@ -3,4 +3,4 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in circuitry.gemspec
 gemspec
 
-gem 'memcache_mock', '0.0.14', github: 'mhuggins/MemcacheMock', branch: 'expiry-and-add'
+gem 'memcache_mock', '0.0.14', git: 'https://github.com/mhuggins/MemcacheMock', branch: 'expiry-and-add'
```

Comment:
@meagerfindings It might be worth bringing this forked change under the @kapost org.  I have no reason to remove or change it, but just adds a level of security to this gem.